infra/base-images/base-runner/coverage

```diff
@@ -111,8 +111,22 @@ function run_fuzz_target {
   fi
 }
 
+export GOPATH=/out/root/go
 # Run each fuzz target, generate raw coverage dumps.
 for fuzz_target in $FUZZ_TARGETS; do
+  # Test if fuzz target is a golang one.
+  if [[ $FUZZING_LANGUAGE == "go" ]]; then
+    # Continue if not a fuzz target.
+    if [[ $FUZZING_ENGINE != "none" ]]; then
+      grep "go test -run" $fuzz_target > /dev/null 2>&1 || continue
+    fi
+    cd $GOPATH/src
+    echo "Running go target $fuzz_target"
+    export FUZZ_CORPUS_DIR="/corpus/${fuzz_target}/"
+    bash $OUT/$fuzz_target $DUMPS_DIR/$fuzz_target.profdata
```

Comment:
The existing code runs fuzz targets in parallel. Can we run Go targets in parallel too?

Reply:
It looks like it works :-)
Refactored the code to share the multiprocess handling